fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchQuantityTest.java

```diff
@@ -170,163 +193,201 @@ public void testSearchQuantity_Quantity_NoDisplayUnit() throws Exception {
         assertSearchDoesntReturnSavedResource("missing-Quantity:missing", "false");
     }
-    
-    /*
-     * Currently, documented in our conformance statement. We do not support
-     * modifiers on chained parameters. https://ibm.github.io/FHIR/Conformance#search-modifiers
-     * Refer to https://github.com/IBM/FHIR/issues/473 to track the issue.
-     */
-    
-//    @Test
-//    public void testSearchQuantity_Quantity_chained_missing() throws Exception {
-//        assertSearchReturnsComposition("subject:Basic.Quantity:missing", "false");
-//        assertSearchDoesntReturnComposition("subject:Basic.Quantity:missing", "true");
-//        
-//        assertSearchReturnsComposition("subject:Basic.missing-Quantity:missing", "true");
-//        assertSearchDoesntReturnComposition("subject:Basic.missing-Quantity:missing", "false");
-//    }
-    
+
+    // Range is 5-10 seconds
     @Test
-    public void testSearchQuantity_Range() throws Exception {
-        // Range is 5-10 seconds
-        
-        // the range of the search value doesn't fully contain the range of the target value
-        assertSearchDoesntReturnSavedResource("Range", "4||s");
-        assertSearchDoesntReturnSavedResource("Range", "5||s");
-        assertSearchDoesntReturnSavedResource("Range", "10||s");
-        assertSearchDoesntReturnSavedResource("Range", "11||s");
-        
+    public void testSearchQuantity_Range_NE() throws Exception {
         assertSearchReturnsSavedResource("Range", "ne4||s");
         assertSearchReturnsSavedResource("Range", "ne5||s");
         assertSearchReturnsSavedResource("Range", "ne10||s");
         assertSearchReturnsSavedResource("Range", "ne11||s");
-        
+    }
+
+    @Test
+    public void testSearchQuantity_Range_AP() throws Exception {
         assertSearchDoesntReturnSavedResource("Range", "ap4||s");
         assertSearchReturnsSavedResource("Range", "ap5||s");
         assertSearchReturnsSavedResource("Range", "ap10||s");
         assertSearchDoesntReturnSavedResource("Range", "ap11||s");
-        
+    }
+
+    @Test
+    public void testSearchQuantity_Range_EQ_Implied() throws Exception {
+        // the range of the search value doesn't fully contain the range of the target value
+        assertSearchDoesntReturnSavedResource("Range", "4||s");
+        assertSearchDoesntReturnSavedResource("Range", "5||s");
+        assertSearchReturnsSavedResource("Range", "10||s");
+        assertSearchDoesntReturnSavedResource("Range", "11||s");
+    }
+
+    @Test
+    public void testSearchQuantity_Range_LT() throws Exception {
         assertSearchDoesntReturnSavedResource("Range", "lt4||s");
         assertSearchDoesntReturnSavedResource("Range", "lt5||s");
-        assertSearchReturnsSavedResource("Range", "lt10||s");
+
+        // Lower Bound is 9.5 for 11, therefore adding precision.
+        // And this is still a RANGE search.
+        assertSearchReturnsSavedResource("Range", "lt10.05||s");
```

Comment:
I suggest restoring this to `lt10||s` ... this should still return the saved resource because the range `lt10` overlaps with the range of the value which is `5 - 10`